html-css/override-extend-variables/step2-instructions.md

```diff
@@ -1,32 +1,36 @@
-## Override PatternFly variables to achieve a new theme
-
-Now that we have a card on our page, let's customize the styles applied to it.
-
-1) <strong>Add a new color variable to use in the card.</strong> Let's make the links in the card purple! In PatternFly, all of the css values we use are first assigned to variables. We recommend doing the same when making customizations in your application.
+There is now a card on the page, customize the styles applied to it.
+ 
+1) <strong>Add a new color variable to use in the card.</strong> 
 
+ Now make the links in the card purple! In PatternFly, all of the CSS values used are first assigned to variables. It is recommended doing the same when making customizations in the application.
+ 
 In `myapp.scss` inside of the `:root` block make a new variable name called `--my-app-card-theme--Color` and assign it to be purple: `#6200EE`.
```

Comment:
Suggested change: In `myapp.scss` inside of the `:root` block, make a new variable name called `--my-app-card-theme--Color`, and assign it to be purple: `#6200EE`. (punctuation fix)